apisix/plugins/jwt-auth.lua

```diff
@@ -28,7 +29,7 @@ local ngx_time = ngx.time
 local sub_str  = string.sub
 local plugin_name = "jwt-auth"
 local pcall = pcall
-
+local jwt_vault_prefix = "jwt-auth/keys/"
```

Comment:
What about changing it to `consumer/<username>/jwt-auth/`?
I think about it several times. Although it requires to change several places in this PR, the new format is more extendable, and easier to recognize. People will remember the username better than the key.

Reply:
Thanks for the suggestion. Addressed